eng/Directory.Build.Common.props

```diff
@@ -3,7 +3,7 @@
   <!-- Setup project type properties -->
   <PropertyGroup>
     <IsMgmtLibrary Condition="'$(IsMgmtLibrary)' == '' and $(MSBuildProjectName.StartsWith('Azure.ResourceManager.'))">true</IsMgmtLibrary>
-    <MgmtAutorestVersion>3.0.0-beta.20210319.1</MgmtAutorestVersion>
+    <MgmtAutorestVersion>3.0.0-beta.20210504.1</MgmtAutorestVersion>
```

Comment:
I believe this will break everyone since it will generate an incomplete API until our work is done

Reply:
The build is green 😄 This is a build of the https://github.com/Azure/autorest.csharp/tree/feature/mgmt-hotfix branch created from the commit before we merged the new management generator.